static/i18n/en/errors.json

```diff
@@ -153,5 +153,9 @@
   },
   "InvalidAddress": {
     "title": "This is not a valid {{currencyName}} address"
+  },
+  "CantOpenDevice": {
+    "title": "Oops, couldn’t connect to device",
+    "description": "Device detected but connection failed. Please retry and get help if the problem persists."
```

Comment:
I don't understand why crowdin bot get lost. We may disable auto sync and trigger it manually after deleting the i10n branch